README.md

```diff
@@ -1,2 +1,246 @@
-# budget-app
-This is a Ruby on Rails capstone project, a mobile web application for managing your budget. It lists categories of transactions and details of each transaction. to show how much money you spent and on what.  Users register and log in to make data private to them. 
+# Budget App: Ometman
+
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="./ometman-logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+  <h3><b>Budget App: by Ometman</b></h3>
+  <h5>Budget App is a beautiful Ruby on Rails mobile web application for managing your budget. Users who must login, can create expense categories and transactions specific to each.  The home shows categories specific to the logged in user and shows total amount from all categories. Clicking a category shows how much money you spent and on what, with specific total in that category.  Data is private to the logged in user. The power of rails is employed to build models, controllers and views that allows easy CRUD funtionality to be performed. Built on a simple Entity Relational Diagram it has a simple UI for a rich UX. 
+  What more? Enjoy it!
+</div>
+
+# 📗 Table of Contents
+
+- [Budget App: Ometman](#budget-app-ometman)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Ometman Budget App ](#-ometman-budget-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install Packages and Run Tests ](#install-packages-and-run-tests-)
+    - [Usage ](#usage-)
+    - [Deployment](#deployment)
+    - [Authors](#authors)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+# 📖 Ometman Budget App <a name="about-project"></a>
+
+**A Demo Video: Introduction to this Project**
+--[Budget Demo](https://www.loom.com/share/ef4cb156194342bc976de0c19a824b58?sid=9e7437f4-f405-46d8-9f76-edcdaa4a61f8)
+
```

Comment:
- [ ] Good job so far on the video, but it would be nice if you could mention a piece of code in your project that you are proud of implementing. In the video presentation, could you show us a piece of code or feature in your code editor and briefly explain what it does and how proud you are of implementing that feature? It's part of the requirement, please redo the video and keep that part in mind at the end of the demo 👍🏽